shader/shared/drawtag.wgsl

```diff
@@ -18,7 +18,7 @@ struct DrawMonoid {
 let DRAWTAG_NOP = 0u;
 let DRAWTAG_FILL_COLOR = 0x44u;
 let DRAWTAG_FILL_LIN_GRADIENT = 0x114u;
-let DRAWTAG_FILL_RAD_GRADIENT = 0x2dcu;
+let DRAWTAG_FILL_RAD_GRADIENT = 0x29cu;
```

Comment:
I was pleasantly surprised to be able to drop a word from info, but also dismayed because I had to modify the constant again :) At some point, I'll get around to replacing these with better aligned bits so they're easier to work with.